src/build-data/buildh.in

```diff
@@ -22,7 +22,7 @@
 
 #define BOTAN_DISTRIBUTION_INFO "%{distribution_info}"
 
-#define BOTAN_INSTALL_PREFIX "%{prefix}"
+#define BOTAN_INSTALL_PREFIX R"(%{prefix})"
```

Comment:
Great! I was thinking a long time about how to fix this issue. (Change necessary for Windows when backslash is used, e.g. `%{prefix}` = C:\Botan)